models/courses.js

```diff
@@ -0,0 +1,19 @@
+const { model, Schema } = require('mongoose');
+
+const courseSchema = new Schema({
+    title: String,
+    videoUrl: String,
+    videoDescription: String,
+    level: String,
+    subject: String,
+    language: String,
+    country: String,
+    createdBy: String,
+    UpdatedAt: String,
```

Comment:
Instead of having updatedAt field, just add { timestamps: true } as second constructor arg